sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -624,10 +624,9 @@ def __init__(self, **kwargs):
     def _from_generated(cls, sentence):
         return cls(
             sentiment=sentence.sentiment,
-            confidence_scores=SentimentConfidenceScores._from_generated(sentence.sentence_scores),  # pylint: disable=protected-access
+            confidence_scores=SentimentConfidenceScores._from_generated(sentence.confidence_scores),  # pylint: disable=protected-access
             grapheme_offset=sentence.offset,
-            grapheme_length=sentence.length,
-            warnings=sentence.warnings,
+            grapheme_length=sentence.length
```

Comment:
Not this particular line, but for this file - I didn't see warnings added / TextAnalyticsWarning model. Is there a bug for this too?

Reply:
i haven't done anything with the new warnings, going to address that in a separate pr for this issue: https://github.com/Azure/azure-sdk-for-python/issues/10036